### ISSUE TITLE
Have you ever been in this situation?

### ISSUE THREAD

**Issue by jiangdapeng1024** [bubbliiiing/yolov4-pytorch#47]
Hello, big guy, may I ask, I use your training garbage classification, the label also changed to garbage, why the end of the training tag inside suddenly more people, cars, bicycles, etc., and the identification is quite accurate, but my training set does not have these pictures, what is this??

**Comment by bubbliiiing** [comment]
maybe u do not change voc_classes.txt(coco_classes.txt)?
